validation/models.py

```diff
@@ -83,6 +87,7 @@ class Phrase(models.Model):
         (LINKED, "Linked"),
         (VALIDATED, "Validated"),
         (REVIEW, "Needs review"),
+        (USER, "User-submitted"),
```

Comment:
I’m not sure what all of these statuses mean or when recordings should move from one status to another. Perhaps some documentation would clarify that?

Reply:
I knew at one point, but now I have no clue again. I agree that it should be clearly outlined how things move from one status to another. I'll add this to my list and properly document it soon.